spec/models/mods_data_spec.rb

```diff
@@ -0,0 +1,114 @@
+require "spec_helper"
```

Comment:
Could this potentially be located under `spec/models/concerns/mods_data_spec.rb`?